application/helpers/SurveyRuntimeHelper.php

```diff
@@ -439,7 +439,7 @@ public function run($surveyid, $args)
 
             $aGroup['showgroupinfo'] = $showgroupinfo_;
             $aGroup['showdescription']  = (!$this->previewquestion && trim($gl['description']) != "" && $showgroupdesc_);
-            $aGroup['description']      = LimeExpressionManager::ProcessString($gl['description'], null, null, 3, 1);
+            $aGroup['description']      = $gl['description'];
```

Comment:
Just over viewing this.
Shouldn't we add description to the `LimeExpressionManager::updateReplacementFields`?

Reply:
I test with , it can be done
But looking at manual : it's not included.

Seems is lost in 3.X since 3.0.

Then it's a new feature :)